package.json

```diff
@@ -79,7 +79,8 @@
     "react-smooth": "^4.0.0",
     "recharts-scale": "^0.4.4",
```

Comment:
If we're merging recharts-scale inside the main package, can we remove the dependency now?

Reply:
didn't add all the code from `recharts-scale` just the parts necessary for the changes for now